tests/data/power_flow/multi-source-with-angle/params.json

```diff
@@ -1,5 +1,5 @@
 {
-  "calculation_method": ["newton_raphson", "linear"],
+  "calculation_method": ["newton_raphson", "iterative_current"],
```

Comment:
This should be working for all three methods

Reply:
Thanks! I should be careful while copy-pasting!